pkg/kubectl/cmd/taint.go

```diff
@@ -40,6 +40,13 @@ import (
 	utiltaints "k8s.io/kubernetes/pkg/util/taints"
 )
 
+const (
+	MODIFIED    = "modified"
```

Comment:
To decide where the right place is, we should decide: what is this describing.

1) If this is describing a kubelet result, then I guess putting them here and making them more descriptive is ok, i.e
```
MODIFICATION="taint was modified"
UNTAINTED="taint was removed"
```
and so on, isolated the semantics, unambiguously to `kubectl`, basically conveying to the user the result of a lower level put/patch/post that may have occured.